book/website/communication/research-objects.md

```diff
@@ -0,0 +1,37 @@
+(cm-ro)=
+# Research Object to capture the Research Life Cycle
+
+## Prerequisites / recommended skill level
+
+## Summary
+
+Research outcomes encompass publications, data, software, bibliographical material and any other resources (such as experimental protocols, standards) that can be potentially useful for conduting the research. 
+
+Research Objects allow to work open by design and share during the research process and not only the research outputs at the end.
```

Comment:
```suggestion
Research Objects allow to work open by design and share during the research process, instead of only sharing the research outputs at the end of a research project.
```